mitodl/micromasters
# Implemented document sent date

#### What are the relevant tickets?
#1115
Fixes #1205 

#### What's this PR do?
Implements the button to update documents being sent, and shows the document date for documents already sent

#### How should this be manually tested?
Change the status of your `FinancialAid` to `pending-docs`. You should see a date picker and a submit button similar to the invision mockups. Pick any date and click 'Submit'. The dashboard will refresh and you will see the updated document date.
